### PR TITLE
more prompts

### PR DESCRIPTION
**Pull request by tioopental**
more prompts, minor changes

### CODE REVIEW

File: src/prompts/neuro.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export default [
   'chat spams Clueless',
   'POG / poggers',
   'chat spams PauseSama',
-  'chat spams RIGGED',
```
Comment (tioopental) [inline src/prompts/neuro.js:65]: no idea what happened here

File: src/prompts/neuro.js [hunk 2]
```diff
@@ -86,5 +86,22 @@ export default [
...
+  'chat spams nuero',
+  'cursed tts sound',
+  'chat spams SCHIZO Alex',
+  'admits she\'s a god',
+  'feel free to ask me anything',
+  'chat prays',
+  'chat shoots Vedal',
+  'mentions Neuro V1',
+  'her blabbering gets faded',
+  'chat spams RIGGED',
```
Comment (tioopental) [inline src/prompts/neuro.js:103]: no idea what happened here

File: src/prompts/neuro.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ export default [
   'heart',
   'gymbag',
   'spams',
-  'backslash spam',
+   //'backslash spam', after debut im positive its not gonna happen anymore (tbd?)
```
Comment (cjmaxik) [inline src/prompts/neuro.js:23]: It does not harm to have more prompts. Also, we do still have merch with backslash spam... Never say "never"?

File: src/prompts/neuro.js [hunk 4]
```diff
@@ -62,7 +62,7 @@ export default [
   'chat spams Clueless',
   'POG / poggers',
   'chat spams PauseSama',
-  'chat spams RIGGED',
+  'chat spams Flushed',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:65]: Please put it back, add new things to the end

File: src/prompts/neuro.js [hunk 5]
```diff
@@ -86,5 +86,22 @@ export default [
...
+  'starts counting',
+  'chat spams nuero',
+  'cursed tts sound',
+  'chat spams SCHIZO Alex',
+  'admits she\'s a god',
+  'feel free to ask me anything',
+  'chat prays',
+  'chat shoots Vedal',
+  'mentions Neuro V1',
+  'her blabbering gets faded',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:102]: Isn't it 1984 basically?

File: src/prompts/neuro.js [hunk 6]
```diff
@@ -86,5 +86,22 @@ export default [
...
+  'chat spams RIGGED',
+  'someone got bingo before you',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:104]: At this time, please do not add meta things. I don't think they belong here for now. Maybe I will add a separate Meta prompt type in the future.

File: src/prompts/neuro.js [hunk 7]
```diff
@@ -86,5 +86,22 @@ export default [
...
+  '#livestream-chat skull 20+',
```
Comment (cjmaxik) [inline src/prompts/neuro.js:105]: At this time, please do not add meta things. I don't think they belong here for now. Maybe I will add a separate Meta prompt type in the future.

File: src/prompts/neuro.js [hunk 8]
```diff
@@ -86,5 +86,22 @@ export default [
...
+  '#livestream-chat star 20+'
```
Comment (cjmaxik) [inline src/prompts/neuro.js:106]: At this time, please do not add meta things. I don't think they belong here for now. Maybe I will add a separate Meta prompt type in the future.

**Review by cjmaxik — CHANGES_REQUESTED** [review]

File: src/prompts/neuro.js (hunk 5 above)
Comment (tioopental) [reply src/prompts/neuro.js:102]: Well 1984 is censoring her out, today she was still going, just muted, so I'd argue it is different

Comment (cjmaxik) [reply src/prompts/neuro.js:102]: Let's have it in our backlog for now. If it happens again, we'll add it, okay?

Comment (tioopental) [reply src/prompts/neuro.js:102]: Okay

File: src/prompts/neuro.js (hunk 3 above)
Comment (tioopental) [reply src/prompts/neuro.js:23]: Fair. I guess we can ask Vedal on monday if he actually censored it

File: src/prompts/neuro.js (hunk 6 above)
Comment (tioopental) [reply src/prompts/neuro.js:104]: Noted

File: src/prompts/neuro.js (hunk 7 above)
Comment (tioopental) [reply src/prompts/neuro.js:105]: Noted

File: src/prompts/neuro.js (hunk 8 above)
Comment (tioopental) [reply src/prompts/neuro.js:106]: Noted

**Pull request merged**